### PR TITLE
gutenberg 0.2.1 (new formula)

### PR DESCRIPTION
**Pull request by bfontaine**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/gutenberg.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    bin.install "target/release/gutenberg"
+    bash_completion.install "completions/gutenberg.bash-completion"
+    zsh_completion.install "completions/_gutenberg"
+    fish_completion.install "completions/gutenberg.fish"
+  end
+
+  test do
+    pipe_output("#{bin}/gutenberg init mysite", "\nn\nn\n")
+    (testpath/"mysite/content/blog/index.md").write <<-EOS.undent
```
Comment (ilovezfs) [inline Formula/gutenberg.rb]: <<~EOS

File: Formula/gutenberg.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    system "cargo", "build", "--release"
+
+    bin.install "target/release/gutenberg"
+    bash_completion.install "completions/gutenberg.bash-completion"
+    zsh_completion.install "completions/_gutenberg"
+    fish_completion.install "completions/gutenberg.fish"
+  end
+
+  test do
+    pipe_output("#{bin}/gutenberg init mysite", "\nn\nn\n")
```
Comment (ilovezfs) [inline Formula/gutenberg.rb]: exit code is not 0?

File: Formula/gutenberg.rb (hunk 1 above)
Comment (bfontaine) [reply Formula/gutenberg.rb]: TIL; thanks. Fixed.

File: Formula/gutenberg.rb (hunk 2 above)
Comment (bfontaine) [reply Formula/gutenberg.rb]: I always forget it’s not the default. Fixed.

File: Formula/gutenberg.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    EOS
+    (testpath/"mysite/templates/page.html").write <<~EOS
+      {{ page.content | safe }}
+    EOS
+
+    cd testpath/"mysite" do
+      system bin/"gutenberg", "build"
+    end
+
+    assert_equal "<p>Hi I'm Homebrew.</p>\n",
```
Comment (ilovezfs) [inline Formula/gutenberg.rb]: I'd drop the \n and use strip instead of chomp

Comment (bfontaine) [reply Formula/gutenberg.rb]: Fixed.

**Review by marvinhagemeister — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks @bfontaine! :rocket:

**Pull request closed without merging**